libguard/guard_file.cpp

```diff
@@ -109,7 +104,7 @@ void GuardFile::erase(const uint64_t pos, const uint64_t len)
     if (len <= 0)
     {
         guard_log(GUARD_ERROR, "Length passed is %d which is not valid", len);
-        throw std::runtime_error("Not a valid length value");
+        throw InvalidEntityPath("Not a valid length value");
```

Comment:
Why throwing `InvalidEntityPath` exception when failed to erase in the guard file?

Reply:
Added InvalidEntry.